### DISCUSSION TITLE
[灌水]以太适应自测结果是这个分数，我能去空洞里搬砖吗？

### DISCUSSION THREAD

**Discussion by RuidaYang** [share121/inter-knot#1064] [其他]
![Uploading 微信图片_20240810214729.jpg…]()

本人学生，高中还没毕业，老爸因为工作原因买了以太适应自测卡，我出于好奇就顺便
也跟着测了一下，结果拿了个80分，老爸当时就从椅子上跳了起来，逼着我重新测了一
次...据说50分是能从事空洞相关工作的及格线，80分是很高的分数了，这是不是意味着
我也可以去空洞里当搬运工了？（bushi）

**Comment by fu-fu404** [comment]
楼主是撞了大运吧？八十分的以太适应可以成为传奇绳匠了，我二叔70分，现在已经是精英绳匠了

**Comment by WORLD-executeME** [comment]
你太小看自己了，上一个测出80分的已经收到墨提斯院的邀请了
